### DISCUSSION TITLE
Library project updated to Uno.Sdk does not create a nuget package

### DISCUSSION THREAD

**Discussion by BrianDT** [unoplatform/uno#16660] [Q&A] [answered]
I have converted an existing library to to 5.2 based on the 5.2 Platform Library Template, and it no longer generates the nuget package.

The project file is basically a 5.2 Platform Library Template with a few extra PackageReferences and these extra statements in the initial PropertyGroup.

```
    <RootNamespace>Vssl.VisualFramework.UIServices</RootNamespace>
    <AssemblyName>Vssl.Framework.UIServices</AssemblyName>
    <PackageId>VSSL.Framework.UIServices</PackageId>
    <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
    <Authors>BDT</Authors>
    <Company>Visual Software Systems Ltd.</Company>
    <Product>VSSL Cross platform Framework</Product>
    <Description>Visual Framework</Description>
    <Version>4.0.1</Version>
    <Copyright>Visual Software Systems Ltd 2012 - 2024.</Copyright>
	  <CodeAnalysisRuleSet>..\..\..\Vssl.ruleset</CodeAnalysisRuleSet>
	  <GenerateDocumentationFile>true</GenerateDocumentationFile>
```

Am I missing something?

**Comment by jeromelaban** [comment, marked as answer]
Thanks for the report, we've noticed this issue, it is being fixed here: https://github.com/unoplatform/uno/pull/16651.

A workarounds is to set `<IsPackable>` to `true`.

**Reply by BrianDT** [reply]
That works, nice simple workaround.
